src/ODE.jl

```diff
@@ -8,9 +8,9 @@ using Compat
 
 ## minimal function export list
 # adaptive non-stiff:
-export ode23, ode45, ode78
+export ode23, ode45, ode78, ode113
 # non-adaptive non-stiff:
-export ode4, ode4ms
+export ode4, ode4ms, ode_am4
```

Comment:
How about we don't export any symbols and make the `ODE.odeXX` convention a standard?